packages/utils/src/tests/analytics.test.js

```diff
@@ -17,3 +17,18 @@ describe('sendEnterpriseTrackEvent', () => {
     });
   });
 });
+
+describe('sendEnterpriseTrackEventWithDelay', () => {
+  it('should call sendTrackEvent with the enterprise uuid in properties', async () => {
+    const mockEnterpriseUUID = '1';
+    const mockEventName = 'event';
+    const mockEventProperties = { property1: 'property 1' };
+
+    await sendEnterpriseTrackEventWithDelay(mockEnterpriseUUID, mockEventName, mockEventProperties);
```

Comment:
[clarification] While this is asserting the underlying `sendTrackEvent` is called, is this testing the delay behavior at all? For example, would we need to call another `jest.fn()` after this `await` to verify it's called after `sendTrackEvent`?